### PR TITLE
Breaking Bricks code in java

### PR DESCRIPTION
**Pull request by raksha009**
Fixes issue: #4588 
Code added for "Chef and Street food "problem in java

@arnavb plz review

### CODE REVIEW

File: code/online_challenges/src/codechef/BRKBRKS/brkbrks.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	    int w3 = s.nextInt();
+	    ArrayList<Integer> arrLL = new ArrayList<>();
+	    arrLL.add(w1);
+	    arrLL.add(w2);
+	    arrLL.add(w3);
+	    int hits = 0;
+	    while (arrLL.size() !=0){
+	        int sum = 0;
+	        while (sum <strength){
+	            sum = sum + arrLL.get(0);
```
Comment (arnavb) [inline code/online_challenges/src/codechef/BRKBRKS/brkbrks.java]:
```suggestion
	            sum += arrLL.get(0);
```

File: code/online_challenges/src/codechef/BRKBRKS/brkbrks.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class brkbks{
+    public static void main (String[] args) throws java.lang.Exception{
```
Comment (arnavb) [inline code/online_challenges/src/codechef/BRKBRKS/brkbrks.java]:
```suggestion
    public static void main (String[] args) {
```

**Comment by raksha009** [comment]
@arnavb i have made the changes. Plz review my PR

**Comment by arnavb** [comment]
@raksha009 

1. Your old code is still present.
2. When reformatting, use 4 space indentation everywhere.
3. You still haven't fixed 2 and 3 from my previous comment.

**Comment by raksha009** [comment]
@arnavb i did those changes. I just forgot to delete my old code

**Comment by raksha009** [comment]
i have updated the PR as you have told.

**Comment by arnavb** [comment]
@raksha009 Change indentation of the file to 4 spaces and change the folder name as well.

**Comment by raksha009** [comment]
@arnavb done the changes (folder name and 4 space indendation done)

**Comment by arnavb** [comment]
@raksha009 I can't modify this PR myself, but the reformatted code should be:

<details>

```java
import java.io.*;
import java.util.*;

class Brkbks {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int t = s.nextInt();

        while (t-- > 0) {
            int strength = s.nextInt();
            int w1 = s.nextInt();
            int w2 = s.nextInt();
            int w3 = s.nextInt();
            ArrayList<Integer> arrLL = new ArrayList<>();
            arrLL.add(w1);
            arrLL.add(w2);
            arrLL.add(w3);
            int hits = 0;

            while (arrLL.size() != 0) {
                int sum = 0;

                // Calculating the sum of widths of bricks
                while (sum < strength) {
                    sum += arrLL.get(0);
                    if (sum <= strength) {
                        arrLL.remove(0);
                    }
                    if (arrLL.size() == 0) {
                        break;
                    }
                }

                // Hits required by Ada to break the given stack of bricks
                hits++;
            }
            System.out.println(hits);
        }
    }

}

/*
TEST CASE

INPUT
3
3 1 2 2
2 1 1 1
3 2 2 1

OUTPUT
2
2
2
*/
```
</details>

Please change your code to match this.

**Comment by raksha009** [comment]
@arnavb i did made the exact changes (4 spaces indentation done). plz have a look in files changed tab. Thanks

**Comment by raksha009** [comment]
@arnavb done the changes.

**Pull request merged**